### ISSUE TITLE
Strange always-true if-statement

### ISSUE THREAD

**Issue by thundergolfer** [pypa/pipenv#3682]
### Issue description

I was investigating some weird action-at-a-distance problem where deleting everything in `~/.local/shared/virtualenvs` changed the output of: 

```
python3.6 -m venv .venv
source .venv/bin/activate
pip install pipenv

PIPENV_PIPFILE=~/path/to/my/Pipfile pipenv sync
pipenv run pipdeptree
```

(Quite weird! But not the concern of this issue)

When I ran into [this bit of `pipenv` source code](https://github.com/pypa/pipenv/blob/1b865647a1ae09957ba414ccbd7b055a65482749/pipenv/utils.py#L1382) 

https://github.com/pypa/pipenv/blob/1b865647a1ae09957ba414ccbd7b055a65482749/pipenv/utils.py#L1382

That's always going to be `True`, so it's doing nothing. Also `Returns the path of a Pipfile` in the docstring should be `Returns the path of a requirements.txt file`. 

Easy things to clean up 🙂

**Comment by frostming** [comment]
Thanks for that, would you like to shoot a PR for this?

**Comment by frostming** [comment]
Closed via #3684
